crates/invoker-impl/src/invocation_task/service_protocol_runner.rs

```diff
@@ -228,10 +228,25 @@ where
         ]);
 
         // Inject OpenTelemetry context
-        TraceContextPropagator::new().inject_context(
-            &Span::current().context(),
-            &mut opentelemetry_http::HeaderInjector(&mut headers),
-        );
+        {
+            let span_context = parent_span_context.span_context();
```

Comment:
Let's write down here one/two sentences about the implication: we now attach to the parent span created by the PP, and not to the invoker span.